Sources/MapboxMaps/Foundation/Extensions/Turf/Feature.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 import MapboxCommon
 
-// MARK: - Feature
+public typealias Feature = Turf.Feature
 
 extension Turf.Feature {
```

Comment:
since you define `Feature` above, can we make this `extension Feature`?

Reply:
Yes, we could clean this up throughout the codebase for Feature and Geometry. We have a bunch of unnecessary `Turf.` say this point. Could you make a ticket to clean them all up at once?